js/WebAPI/DocumentEditor.md

```diff
@@ -1,20 +1,20 @@
 ---
 layout: post
-title: webAPI reference for DocumentEditor
-description: webAPI reference for 
+title: WebAPI reference in JavaScript DocumentEditor Control | Syncfusion
+description: Learn here all about webAPI reference support in Syncfusion JavaScript DocumentEditor control, its elements and more.
```

Comment:
Update like - Learn here all about WebAPI reference support in Syncfusion JavaScript Document Editor control, its elements, and more.

Reply:
resolved